spec/models/miq_report_spec.rb

```diff
@@ -336,6 +336,13 @@
     expect(after.table).to eq(fake_ruport_data_table)
   end
 
+  it "serializes (marshall/dump) special columns" do
+    before = MiqReport.new.tap { |r| r.extras = {:working => true} }
+    # reports are put into session[:view]. ensure when it comes out of the session, we don't loose values
```

Comment:
If we're fixing other thing, might as well fix this typo too.

```suggestion
    # reports are put into session[:view]. ensure when it comes out of the session, we don't lose values
```